pkgs/development/tools/rust/cargo-local-registry/default.nix

```diff
@@ -0,0 +1,53 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, pkg-config
+, curl
+, openssl
+, zlib
+, stdenv
+, darwin
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cargo-local-registry";
+  version = "0.2.3";
+
+  src = fetchFromGitHub {
+    owner = "dhovart";
+    repo = "cargo-local-registry";
+    rev = version;
+    hash = "sha256-nxLqWtZl3ZF/iodYsQCYQ/prjp80QMzJLLp31q7d2vs=";
+  };
+
+  cargoHash = "sha256-k94jzMdZDWpxSHVEZh1Qsv8OuUKuqU2YNBN1Mqj8HJA=";
+
+  nativeBuildInputs = [
+    curl
+    pkg-config
+  ];
+
+  buildInputs = [
+    curl
+    openssl
+    zlib
```

Comment:
This was generated with nix-init since the package depends on libz-sys, without this the crate can fall back to its vendored version of zlib. I'm not sure if I should add a comment for this since this is a very common system dependency